### DISCUSSION TITLE
Find the full names of authors who cite my papers.

### DISCUSSION THREAD

**Discussion by RaptorMai** [scholarly-python-package/scholarly#533] [Q&A]
Is that possible to find the full names of the authors who cite my papers? It seems the current version only supports [first letter of first name, full last name]. Also, is that possible to find those authors' affiliations?

**Comment by arunkannawadi** [comment]
Unfortunately, no. Google Scholar reports that results in that format, and that limits what information you can get out of `scholarly` package as well. If you can find a page in the browser version where the citation list has the full names of the authors, post the link here and we can work towards incorporating that feature.
